tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -181,6 +181,37 @@ def construct(self):
         self.add(im1, im2, im3)
         self.wait(1)
 
+class ImageInterpolationTest(Scene):
+    def construct(self):
+        image1 = ImageMobject(np.uint8([[63, 0, 0, 0],
+                                        [0, 127, 0, 0],
+                                        [0, 0, 191, 0],
+                                        [0, 0, 0, 255]
+                                        ]), scale_to_resolution=7)
+        image2 = ImageMobject(np.uint8([[63, 0, 0, 0],
+                                        [0, 127, 0, 0],
+                                        [0, 0, 191, 0],
+                                        [0, 0, 0, 255]
+                                        ]), scale_to_resolution=7)
+        image1.height = 7
+        image2.height = 7
+        image1.set_interpolation_algorithm("nearest")
+        self.add(image1)
+        image2.set_interpolation_algorithm("antialias")
+        self.play(FadeIn(image2), run_time=0.2)
+        self.remove(image1)
+        image1.set_interpolation_algorithm("linear")
+        self.play(FadeIn(image1), run_time=0.2)
+        self.remove(image2)
+        image2.set_interpolation_algorithm("cubic")
+        self.play(FadeIn(image2), run_time=0.2)
+        self.remove(image1)
+        image1.set_interpolation_algorithm("box")
+        self.play(FadeIn(image1), run_time=0.2)
+        self.remove(image2)
+        self.wait()
```

Comment:
**Issue :** Unfortunatly, this won't test anyting, since only the last frame is compared for the test (at the moment). 

You've to do separate tests for that. 

Thanks for thinking to implement tests nevertheless !

Reply:
> I don't think it's a good think to let interpolation_algorithmparamter be a either a str or a an int.

My motivation for keeping this duality was to leave an option to the users of manim not to import the Pillow and keep the public dependencies as small as possible (I guess not everyone would expect the need to import Pillow in order to use manim, the documentation never mentions that anyway). If manim was written in C++ I would never do that, and manim would be published along with its headers, where those constants would be re-published.

If you can live with this duality of types, I would be grateful if you let me keep them.

Reply:
>Unfortunatly, this won't test anyting, since only the last frame is compared for the test (at the moment).
How does one writes parametrized tests?

I would add a test like this

```py
class ImageInterpolationTest(Scene):
    def construct(self, alg):
        image1 = ImageMobject(np.uint8([[63, 0, 0, 0],
                                        [0, 127, 0, 0],
                                        [0, 0, 191, 0],
                                        [0, 0, 0, 255]
                                        ]), scale_to_resolution=7)
        image1.height = 7
        image1.set_interpolation_algorithm(alg)
        self.add(image1)
        self.wait(1)

```

...and let the pytest call it for all eligible values of the parameter `alg`. How does one do that?

Reply:
That's unfortunatly not possible to do that with the current implementation of the graphical unit tests :/